react-native-libraries.json

```
@@ -5810,7 +5810,8 @@
     "web": true,
     "expo": true,
     "windows": true,
-    "macos": true
+    "macos": true,
+    "examples": ["https://github.com/dcangulo/react-native-pkce-challenge/tree/main/example"],
```

Hey @dcangulo, thanks for the updates! 👍 You need to remove trailing commas to fix the CI, since they are not valid in JSON files.

```suggestion
    "examples": ["https://github.com/dcangulo/react-native-pkce-challenge/tree/main/example"]
```